### PR TITLE
Hide the inquire button on inquirable works that are for sale

### LINKED ISSUE #857: Remove "Ask a Specialist on 'not for sale'; 'sold'; 'on hold' works.
image

### PR DESCRIPTION
**Pull request by orta**
Fixes #857

The single-line-ification was spacecommander, I don't have a  strong opinion on them.

### CODE REVIEW

**Comment by orta** [comment]
Assigning to @alloy - he might have seen enough of the additional context around the issue

**Comment by orta** [comment]
screen shot 2015-12-18 at 14 25 28

**Comment by orta** [comment]
CHANGELOG entry is in #990

File: Artsy/Views/Artwork/ARArtworkActionsView.m [hunk 1]
```diff
@@ -337,29 +337,22 @@ - (void)setEnabled:(BOOL)enabled
...
-           && self.artwork.isPriceHidden.boolValue;
+    return self.artwork.availability == ARArtworkAvailabilityForSale && self.artwork.isPriceHidden.boolValue;
 }
 
 - (BOOL)showContactButton
 {
-    return self.artwork.forSale.boolValue
-           && !self.artwork.acquireable.boolValue
-           && ![self showAuctionControls];
+    return self.artwork.forSale.boolValue && !self.artwork.acquireable.boolValue && ![self showAuctionControls];
```
Comment (alloy) [inline Artsy/Views/Artwork/ARArtworkActionsView.m]: This last one could also use dot-notation.

**Comment by alloy** [comment]
Ace :+1:

**Pull request merged**